src/auth/dto/create-user.dto.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  IsString,
+  MinLength,
+  Matches,
+  IsNotEmpty,
+  IsEmail,
+  Length,
+} from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+export class CreateUserDto {
+  @IsString()
+  @ApiProperty({
+    example: 'username',
```

Comment:
PullProofAI

The `name` field is missing a length validation. Consider adding a `@Length` decorator to ensure the length of the name falls within a specified range.

```diff
+   @Length(2, 50, {
+     message: 'Name should be between 2 and 50 characters',
+   })
```



<!-- This is an auto-generated comment by PullProofAI -->